config/mongoid.yml

```diff
@@ -1,19 +1,143 @@
+      # of host:port pairs. (required)
+      hosts:
+        - localhost:27017
+      options:
+        # Change whether the session persists in safe mode by default.
+        # (default: false)
+        # safe: false
+
+        # Change the default consistency model to :eventual or :strong.
+        # :eventual will send reads to secondaries, :strong sends everything
+        # to master. (default: :eventual)
+        # consistency: :eventual
+
+        # How many times Moped should attempt to retry an operation after
+        # failure. (default: 30)
+        # max_retries: 30
+
+        # The time in seconds that Moped should wait before retrying an
+        # operation on failure. (default: 1)
+        # retry_interval: 1
+  # Configure Mongoid specific options. (optional)
+  options:
+    # Configuration for whether or not to allow access to fields that do
+    # not have a field definition on the model. (default: true)
+    # allow_dynamic_fields: true
+
+    # Enable the identity map, needed for eager loading. (default: false)
+    # identity_map_enabled: false
+
+    # Includes the root model name in json serialization. (default: false)
+    # include_root_in_json: false
+
+    # Include the _type field in serializaion. (default: false)
+    # include_type_for_serialization: false
+
+    # Preload all models in development, needed when models use
+    # inheritance. (default: false)
+    # preload_models: false
+
+    # Protect id and type from mass assignment. (default: true)
+    # protect_sensitive_fields: true
+
+    # Raise an error when performing a #find and the document is not found.
+    # (default: true)
+    # raise_not_found_error: true
+
+    # Raise an error when defining a scope with the same name as an
+    # existing method. (default: false)
+    # scope_overwrite_exception: false
+
+    # Skip the database version check, used when connecting to a db without
+    # admin access. (default: false)
+    # skip_version_check: false
+
+    # User Active Support's time zone in conversions. (default: true)
+    # use_activesupport_time_zone: true
+
+    # Ensure all times are UTC in the app side. (default: false)
+    # use_utc: false
 
 test:
-  host: localhost
-  database: tomatoes_app_test
-  use_activesupport_time_zone: true
+  sessions:
+    default:
+      database: tomatoes_app_test
+      hosts:
+        - localhost:27017
+      options:
+        consistency: :strong
+        # In the test environment we lower the retries and retry interval to
+        # low amounts for fast failures.
+        max_retries: 1
+        retry_interval: 0
 
-# set these environment variables on your prod server
 production:
-  uri: <%= ENV['MONGOHQ_BIG_URL'] %>
-  use_activesupport_time_zone: true
-  # slaves:
-  #   - host: slave1.local
-  #     port: 27018
-  #   - host: slave2.local
-  #     port: 27019
+  sessions:
+    default:
+      uri: <%= ENV["MONGOHQ_REPL_URI"]%>
```

Comment:
Pay attention that in the [configuration](https://mongoid.github.io/old/en/mongoid/docs/installation.html#configuration) docs they require at least 3 host to run a replica set...

```
# Tell Mongoid which environment this configuration is for.
production:
  # This starts the session configuration settings. You may have as
  # many sessions as you like, but you must have at least 1 named
  # 'default'.
  sessions:
    # Define the default session.
    default:
      # A session can have any number of hosts. Usually 1 for a single
      # server setup, and at least 3 for a replica set. Hosts must be
      # an array of host:port pairs. This session is single server.
      hosts:
        - flame.mongohq.com:27017
      # Define the default database name.
      database: mongoid
      # Since this database points at a session connected to MongoHQ, we must
      # provide the authentication details.
      username: user
      password: password
    # This defines a secondary session at a replica set.
    replica_set:
      # This configuration is a 3 node replica set.
      hosts:
        - dedicated1.myapp.com:27017
        - dedicated2.myapp.com:27017
        - dedicated3.myapp.com:27017
      database: mongoid
      # We can set session specific options, like reads executing
      # on secondary nodes, and defaulting the session to safe mode.
      options:
        read: :secondary
        write:
          w: :majority
```